docs/_packages/caniuse.md

```diff
@@ -0,0 +1,8 @@
+---
+top:         true
+type:        plugin
+title:       "caniuse"
```

Comment:
This is an awesome plugin! Should the title be `Can I use` to match their website?